po/ru.po

```diff
@@ -8,14 +8,15 @@ msgstr ""
 "Project-Id-Version: spot\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-03-16 19:27+0100\n"
-"PO-Revision-Date: 2021-03-16 19:21+0100\n"
+"PO-Revision-Date: 2021-05-5 16:21+0100\n"
```

Comment:
ISO8601 requires that you write the date as 2021-05-05, not 2021-05-5

Reply:
Should be resolved, now.